functions/__tests__/functions.test.ts

```diff
@@ -0,0 +1,134 @@
+import * as functionsTestInit from 'firebase-functions-test'
+import mockedEnv from 'mocked-env'
+import { mocked } from 'ts-jest/utils'
+import { MeiliSearch } from 'meilisearch'
+
+jest.mock('meilisearch')
+
+const defaultEnvironment = {
+  LOCATION: 'us-central1',
+  PROJECT_ID: 'fake-project',
+  COLLECTION_PATH: 'example',
+  INPUT_FIELD_NAME: '',
+  MEILISEARCH_INDEX_NAME: 'example',
+  MEILISEARCH_HOST: 'http://127.0.0.1:7700',
+  MEILISEARCH_API_KEY: 'masterKey',
+}
+
+export const mockExport = (document, data) => {
+  const ref = require('../src/index').indexingWorker
+  const functionsTest = functionsTestInit()
+
+  const wrapped = functionsTest.wrap(ref)
+  return wrapped(document, data)
+}
+
+let restoreEnv
+const functionsTest = functionsTestInit()
+
+describe('extension', () => {
```

Comment:
`mockExport`  could be added in the describe function like the other variables used by the tests